projects/example/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+cd projects/example/my-api-repo
+make clean  # Not strictly necessary, since we are building in fresh dir.
+make -j$(nproc) all    # Build the fuzz targets.
+make -j$(nproc) check  # Sanity check, not strictly required, but nice to have.
+
+# Copy the fuzzer executables and zip-ed corpora to $OUT
+find . -name '*_fuzzer' -exec cp -v '{}' $OUT ';'
+find . -name '*_fuzzer_seed_corpus.zip' -exec cp -v '{}' $OUT ';'
```

Comment:
maybe for a future PR, but should we add an example of .options files and add a line to copy them to $OUT here too?